docs/kubernetes/kctlr-rewrite-url.rst

```diff
@@ -0,0 +1,221 @@
+:version: 1.5
+:type: tutorial
+
+.. _k8s url rewrite:
+
+Rewrite URLs
+============
+
+Overview
+--------
+
+.. include:: /_static/reuse/k8s-version-added-1_5.rst
+
+You can use the |kctlr| to rewrite URLs for Kubernetes Ingress resources and OpenShift Ingresses and Routes.
+
+The Annotations are the same in both orchestration systems. The examples provided in this document are Kubernetes Ingress Resources. Their usage is no different in OpenShift Ingresses or Routes.
+
+.. important::
+
+   If you use both of the virtual-server.f5.com :code:`rewrite` Annotations for the same Service, the Controller applies them in the following order:
+
+   #. :code:`app-root`
+   #. :code:`target-url`
+
+Usage
+-----
+
+You can add the :code:`rewrite` Annotations to any of the following resources:
+
+- Kubernetes Ingress
+- OpenShift Ingress
+- OpenShift Route
+
+.. seealso::
+
+   - :ref:`create k8s ingress`
+   - :ref:`create os route`
+
+When using the Annotation(s) in a Single Service Ingress, Multi-Service Ingress, or Route resource, you can specify a **single value** in the annotation.
+
+When using the Annotation(s) in a Multi-Service Ingress, you can specify a single value --OR-- **multiple, comma-separated** values.
+
+Examples
+--------
+
+Rewrite App-Root
+````````````````
+
+.. sidebar:: Applies to:
+
+   - Kubernetes Single-Service Ingress
+   - Kubernetes Multi-Service Ingress
+   - OpenShift Single-Service Ingress
+   - OpenShift Multi-Service Ingress
+   - OpenShift Route
+
+Use the :code:`rewrite-app-root` Annotation to redirect requests for "/" to a different path.
+
+.. code-block:: YAML
+   :caption: Redirect "/" to "/home" for all traffic
+
+   apiVersion: extensions/v1beta1
+   kind: Ingress
+   metadata:
+     annotations:
+       virtual-server.f5.com/rewrite-app-root: "/home"
+     name: home
+     namespace: default
+   spec:
+     rules:
+     - http:
+       paths:
+       - backend:
+           serviceName: http-svc
+           servicePort: 80
+           path: /home
+
+.. code-block:: YAML
+   :caption: Rewrite "/" to "/en" for traffic with host "docs.example.com"
+
+   apiVersion: extensions/v1beta1
+   kind: Ingress
+   metadata:
+     annotations:
+       virtual-server.f5.com/rewrite-app-root: "docs.example.com=/en"
+     name: docs-rewrite
+     namespace: default
+   spec:
+     rules:
+     - host: docs.example.com
+       http:
+         paths:
+         - backend:
+             serviceName: svc-docs
+             servicePort: 80
+             path: /docs-en
+     - host: example.com
+       http:
+         paths:
+         - backend:
+             serviceName: svc-home
+             servicePort: 80
+             path: /home
+
+.. code-block:: YAML
+   :caption: Rewrite multiple paths in a single annotation
+
+   apiVersion: extensions/v1beta1
+   kind: Ingress
+   metadata:
+     annotations:
+       virtual-server.f5.com/rewrite-app-root: "example.com=/home,docs.example.com=docs.example.com/en"
```

Comment:
docs.example.com=docs.example.com/en -> docs.example.com=/en since you are not reseting the host you do not need to set it on the right hand side of the = just the path needs to be set since that is the only thing you are rewriting.